### PR TITLE
updated serializers , viewsets and urls.py

### PR DESCRIPTION
**Pull request by asmitagupta**
Added few more classes to the serializers and also the related viewswets .

### CODE REVIEW

**Comment by dmorina** [comment]
accept the CLA please

File: csp/urls.py [hunk 1]
```diff
@@ -15,7 +17,16 @@
 router.register(r'api/project', ProjectViewSet)
 router.register(r'api/category', CategoryViewSet)
 router.register(r'api/module', ModuleViewSet)
-router.register(r'api/projectrequester', ProjectRequesterViewSet)
+router.register(r'api/projectrequester', ProjectRequesterViewSet)router.register(r'api/WorkerSkill', WorkerSkillViewSet)
```
Comment (dmorina) [inline csp/urls.py]: new line?

File: csp/urls.py [hunk 2]
```diff
@@ -15,7 +17,16 @@
...
+router.register(r'api/Worker', WorkerViewSet)
```
Comment (dmorina) [inline csp/urls.py]: use lower case and separate words with  a dash

**Comment by nistala** [comment]
@dmorina LGTM from my end, can you send your LGTM as well?

**Comment by asmitagupta** [comment]
i have added one more class so thats it nothing else to add

**Pull request merged**